packages/core/src/domain/configuration/transportConfiguration.ts

```diff
@@ -40,8 +35,9 @@ export function computeTransportConfiguration(
   }
 }
 
-function computeEndpointBuilders(initConfiguration: InitConfiguration, buildEnv: BuildEnv, tags: string[]) {
-  if (buildEnv.buildMode === BuildMode.E2E_TEST) {
+function computeEndpointBuilders(initConfiguration: InitConfiguration, tags: string[]) {
+  // @ts-ignore replaced at build time
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** to avoid those ts-ignore, maybe add something like this in each module where you are using those kind of constants:

```ts
// replaced at build time
declare const __BUILD_ENV__BUILD_MODE__: string
```